.github/workflows/ci-windows.yml

```diff
@@ -0,0 +1,48 @@
+name: Windows CI
+on: [pull_request]
+
+jobs:
+  run-windows-tests:
+    name: Build & run tests
+    runs-on: windows-2022
+
+    steps:
+    - uses: actions/checkout@v3
+      name: Checkout Code
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '^14'
+        cache: 'yarn'
+
+    - name: Setup MSBuild
+      uses: microsoft/setup-msbuild@v1.1.3
+      with:
+        vs-version: '[17.0,)'
+        msbuild-architecture: x64
+
+    - name: Install node modules
+      run: yarn --pure-lockfile
+
+    - name: yarn build
+      run: |
+        yarn build
+        yarn tsc
+
+    - name: Build x64 release
+      shell: powershell
+      run: yarn windows --release --no-packager --no-deploy --logging
+
+    # Workaround for a bug in package searching during deploy.

```

Comment:
If it is a known issue, please add a link to it here so that in the future, if anyone wants to refactor, they can check the link for status updates

Reply:
I didn't pay attention to the comments; I have already removed them as they were not necessary.